ENIGMAsystem/SHELL/Audio_Systems/SDL/SDLadvanced.cpp

```diff
@@ -1,12 +1,12 @@
-#include <stdio.h>
-#include <string>
+#include "SoundResource.h"
+#include "SoundChannel.h"
+#include "SDLsystem.h"
 #include "../General/ASadvanced.h"
 #include "Audio_Systems/audio_mandatory.h"
 #include "Widget_Systems/widgets_mandatory.h"
 #include "../General/ASutil.h"
-#include "SoundResource.h"
-#include "SoundChannel.h"
-#include "SDLsystem.h"
+#include <string>
+#include <stdio.h>
 #include <SDL_mixer.h>

```

Comment:
Put mixer first over here too:
```suggestion
#include <SDL_mixer.h>
#include <string>
#include <stdio.h>
```